apps/studio/prisma/generated/generatedTypes.ts

```diff
@@ -8,7 +8,7 @@ export type Generated<T> =
     : ColumnType<T, T | undefined, T>
 export type Timestamp = ColumnType<Date, Date | string, Date | string>
 
-export interface Blob {
+export type Blob = {
```

Comment:
Why do we always have this conflict? Sometimes it generates interfaces, sometimes types

Reply:
i honestly have no clue